### PR TITLE
Migrations: Make first-round API Review changes

### PR DESCRIPTION
**Pull request by bricelam**
Lots of renames and a few minor refactorings...

### CODE REVIEW

File: src/EntityFramework.Commands/Migrations/CSharpMigrationOperationGenerator.cs [hunk 1]
```diff
@@ -307,7 +303,7 @@ protected virtual void Generate([NotNull] AlterColumnOperation operation, [NotNu
                 }
 
                 builder.AppendLine(",")
-                    .Append("nullable: ")
+                    .Append("isNullable: ")
```
Comment (AndriySvyryd) [inline src/EntityFramework.Commands/Migrations/CSharpMigrationOperationGenerator.cs]: Couldn't these strings be replaced by nameof()?

Comment (anpete) [inline src/EntityFramework.Commands/Migrations/CSharpMigrationOperationGenerator.cs]: Likely only in the scope of the owning method.

Comment (anpete) [inline src/EntityFramework.Commands/Migrations/CSharpMigrationOperationGenerator.cs]: "isNullable" should be "nullable" to be idiomatic.

**Comment by AndriySvyryd** [comment]
:shipit:

File: src/EntityFramework.Commands/Migrations/CSharpMigrationOperationGenerator.cs (hunk 1 above)
Comment (bricelam) [inline src/EntityFramework.Commands/Migrations/CSharpMigrationOperationGenerator.cs]: @anpete We have *is* prefixes on Fluent API parameters:
* IndexBuilder
    * Unique(isUnique)
    * SqlServerClustered(isClustered)
* PropertyBuilder
    * Required(isRequired)
    * ConcurrencyToken(isConcurrencyToken)
* ReferenceCollectionBuilder/ReferenceReferenceBuilder
    * Required(isRequired)
* KeyBuilder
    * SqlServerClustered(isClustered)

I'll leave for now, and we can discuss in the next API Review.

**Pull request merged**